general/community/contribute/new-features-ideas.md

```diff
@@ -0,0 +1,34 @@
+---
+title: New features and ideas
+tags:
+- Core development
+- Processes
+- Community
+---
+
+## Where to begin
+
+- Start by [searching moodle.org](http://moodle.org/public/search/) to check whether someone else has had the same idea. The Moodle community is very large, so it's quite likely you'll find someone else with the same idea as you ;-)
+- Either join an existing discussion about your idea or start a new discussion in an appropriate [forum on moodle.org](http://moodle.org/course/view.php?id=5).
+- Assuming others agree with your idea, [create a new issue in the Moodle tracker](http://tracker.moodle.org/secure/CreateIssue!default.jspa), selecting 'New feature' or 'Improvement' as the issue type. (You'll need to create a tracker account to be able to create a new issue.) Include a link to the discussion thread in the tracker issue.
+- Post the tracker issue number in the discussion thread, to encourage others to watch, vote, comment and/or come up with a patch for it.
+
+## What happens next?
+
+- The [list of most voted-for new features](http://tracker.moodle.org/secure/IssueNavigator.jspa?mode=hide&requestId=10512) is regularly reviewed by Moodle HQ and other core developers.
+- Promising ideas are added to the Roadmap (depending on funding availability).
+- Small feature ideas and improvements may be added to core by the component maintainer.
+
+## How to maximize the chance of your idea being implemented
+
+Ideas with lots of votes are more likely to be implemented, however votes are not the only deciding factor.
+
+- If you're able to provide funding for your idea to be implemented, please contact a [Moodle Partner specializing in custom development](http://moodle.com/partners/).
+- Alternatively, you could join the [Moodle Users Association](https://moodleassociation.org/) and propose your idea there.
+- If you're a developer and can create a patch for your feature idea or improvement,
+  please attach it to the tracker issue for review by the component maintainer. When you add a patch, add a "patch" label so your patch will be found in searches.
+  The M in Moodle stands for modular, and so the easiest ideas for new developers to implement are new modules and plugins.
```

Comment:
Take this line out - it's not true. Moodle has not been an acronym for 15 years. Perhaps re-word it?